tests/Remove-DbaDbView.Tests.ps1

```diff
@@ -0,0 +1,48 @@
+$CommandName = $MyInvocation.MyCommand.Name.Replace(".Tests.ps1", "")
+Write-Host -Object "Running $PSCommandPath" -ForegroundColor Cyan
+. "$PSScriptRoot\constants.ps1"
+
+Describe "$CommandName Unit Tests" -Tag 'UnitTests' {
+    Context "Validate parameters" {
+        [array]$params = ([Management.Automation.CommandMetaData]$ExecutionContext.SessionState.InvokeCommand.GetCommand($CommandName, 'Function')).Parameters.Keys
+        [object[]]$knownParameters = 'SqlInstance', 'SqlCredential', 'Database', 'View', 'InputObject', 'EnableException'
+        It "Should only contain our specific parameters" {
+            Compare-Object -ReferenceObject $knownParameters -DifferenceObject $params | Should -BeNullOrEmpty
+        }
+    }
+}
+
+Describe "$CommandName Integration Tests" -Tag "IntegrationTests" {
+
+    BeforeAll {
+
+        $instance2 = Connect-DbaInstance -SqlInstance $script:instance2
+        $null = Get-DbaProcess -SqlInstance $instance2 | Where-Object Program -match dbatools | Stop-DbaProcess -Confirm:$false
```

Comment:
I think this line should be removed - as tests migth run in parallel.

Reply:
This was added by me because it helps with appveyor's limited resources (tho the new experimental path may help with exhaustion). Tests do not run in parallel as they are isolated to synchronous commands in a VM. Please leave it in. 

Also, I'd like to prioritize the merging of this PR as it may cause burnout in the author and he used my commands as a baseline 🤗

Reply:
Thanks for the info. So no change needed here.